.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - '**.rst'
+      - 'docs/**'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.rst'
+      - 'docs/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+  
+jobs:
+  build_wheels:
+    strategy:
+      matrix:
+        os: ["ubuntu-20.04", "macos-11"]
```

Comment:
Maybe use latest?

```suggestion
        os: ["ubuntu-latest", "macos-latest"]
```

Reply:
I picked the oldest ones for more compatible toolchains, but this may be obviated by the containers used by `cibuildwheel`.